packages/cli/src/commands/link/linkAll.js

```diff
@@ -1,64 +1,53 @@
-// @flow
+/**
+ * @flow
+ */
 
-import {uniqBy, flatten} from 'lodash';
+import {uniqBy, flatMap} from 'lodash';
 import path from 'path';
-import type {
-  ContextT,
-  PlatformsT,
-  ProjectConfigT,
-} from '../../tools/types.flow';
-import logger from '../../tools/logger';
-import getAssets from '../../tools/getAssets';
-import getProjectDependencies from './getProjectDependencies';
-import getDependencyConfig from './getDependencyConfig';
+import dedent from 'dedent';
+
+import type {ConfigT, PlatformsT} from '../../tools/config/types.flow';
+
 import promiseWaterfall from './promiseWaterfall';
 import commandStub from './commandStub';
 import promisify from './promisify';
+
 import linkAssets from './linkAssets';
 import linkDependency from './linkDependency';
 
 const dedupeAssets = (assets: Array<string>): Array<string> =>
   uniqBy(assets, asset => path.basename(asset));
 
-function linkAll(
-  context: ContextT,
-  platforms: PlatformsT,
-  project: ProjectConfigT,
-) {
-  logger.warn(
-    'Running `react-native link` without package name is deprecated and will be removed ' +
```

Comment:
We decided not to deprecate it, since whole `link` will become obsolete at some point.